src/components/Request/FormFields/LinkedinSearchTitles.tsx

```diff
@@ -0,0 +1,123 @@
+import { Input, Text } from "@fluentui/react-components";
+import { Controller } from "react-hook-form";
+import { FormField } from "components/Request/NewRequestForm";
+import "components/Request/Request.css";
+import { TextFieldIcon } from "@fluentui/react-icons-mdl2";
+
+const rules = {
+  maxLength: {
+    value: 255,
+    message: "Job Titles can be no longer than 255 characters",
+  },
+};
+
+const LinkedinSearchTitles = ({ name, form }: FormField) => {
+  return (
+    <fieldset>
+      <legend>
+        <TextFieldIcon className="requestFieldIcon" />
+        <Text weight="semibold">Similar Job Titles to Search For</Text>
```

Comment:
Maybe stay consistent with the others and drop "to Search For" as these are all criteria to search for
```suggestion
        <Text weight="semibold">Similar Job Titles</Text>
```